docs/en/observability/create-alerts.asciidoc

```diff
@@ -7,6 +7,10 @@ Make sure alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
+//REVIEWERS: What should I do with references to the Uptime app?
+//What about the alerting rules related to Uptime?
+//Uptime no longer appears in the UI (by default, at least).
```

Comment:
Commented in the other PR, but we should leave these. It doesn't show up by default for new users without heartbeat data, but it is still available for existing users with heartbeat data.